webui/src/components/KeyResults.vue

```diff
@@ -77,8 +101,16 @@
                     objective: objective,
                     keyresult: keyresult,
                     krStatement: keyresult.statement,
-                    krPercent: keyresult.percent
+                    krPercent: keyresult.percent,
+                    krDescription: keyresult.description
                 });
+            },
+
+            openChat(keyresultId) {
+                this.showDescription = !this.showDescription;
+
+                // TODO : FIND InputForm html-element WHERE :name == keyresultId
```

Comment:
@preigile the idea is to dynamically change element style.
But I think it is Ok to merge even current functionality for some time.
Will fix it later if it is not easy to find a fix right now.